application-analytics-ui/src/main/resources/Analytics/Code/Macros/Configurations.xml

```diff
@@ -20,468 +20,523 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
 
-<xwikidoc version="1.4" reference="Analytics.Code.Macros.Configurations" locale="">
```

Comment:
Why there are so many changes on this page? I don't see a comment on the PR description about this. Maybe you exported an older version of the page, or it is from a newer version of XWiki?

Reply:
I added the wrong version of the file